src/test/java/IMDbTop250Scraper.java

```diff
@@ -0,0 +1,92 @@
+import com.codeborne.selenide.*;
+ import java.util.ArrayList;
+import java.util.List;
+import static com.codeborne.selenide.Selenide.*;
+
+public class IMDbTop250Scraper {
+
+    public static void main(String[] args) {
+        configureSelenide();
+        openImdbPage();
+        openNavigationMenu();
+        navigateToTop250();
+        List<String> movieData = scrapeMovies(20);
+        printTopMovies(movieData);
+        double averageRating = calculateAverageRating(movieData);
+        printAverageRating(averageRating);
+    }
+
+    private static void configureSelenide() {
+        Configuration.timeout = 10000;
+    }
+    private static void openImdbPage() {
+        open("https://www.imdb.com");
+    }
+    private static void openNavigationMenu() {
+        $("#imdbHeader-navDrawerOpen").should(Condition.exist).click();
+    }
+    private static void navigateToTop250() {
+        SelenideElement top250Link = $("aside.drawer a:nth-child(2)");
```

Comment:
Again... you are using flaky selectors like a:nth-child(2), what if tomorrow it will be a:nth-child(4) or a:nth-child(7).
So as you can see, this is a link text, so we can use that. $("[href='/chart/top/?ref_=nv_mv_250']");

Like in the previous Pull Request you can do chain linking you know element.shouldBe(visible)...